<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerWindow</class>
 <widget class="QMainWindow" name="ServerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1002</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ServerWindow</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>291</width>
      <height>271</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>60</y>
      <width>641</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="port">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>8000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8080</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9000</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9080</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Logs:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Connected Socket Infomation Monitor:</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>761</width>
      <height>51</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QPushButton" name="send_btn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>10</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>491</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Send broadcast message to all clients here</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCommandLinkButton" name="listen_btn">
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="text">
        <string>Listen</string>
       </property>
       <property name="description">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCommandLinkButton" name="stop_btn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1002</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
